.stylelintrc

```diff
@@ -1,7 +1,7 @@
 {
 	"plugins": ["@signal-noise/stylelint-scales"],
 	"rules": {
-		"scales/font-weight": [[ 400, 600, "normal", "bold" ]],
+		"scales/font-weight": [[ 400, 600, 700, "normal", "bold" ]],
```

Comment:
If we're allowing both 600 and 700, how will folks know when to use 600 vs 700?

400 == normal
700 == bold

Reply:
> If we're allowing both 600 and 700, how will folks know when to use 600 vs 700?

I need to investigate setting up compound rules (ie. only use 700 when using `$serif`). I *think* it's possible, I just need to look at the plugin syntax.

We have plans to look at standardizing typography on the whole, though, so I'm OK with letting a few 700's slip through until we've made more concrete decisions around what fonts/sizes we use.

Reply:
Gotcha, overall code changes look good, so if folks were okay with the style updates, I think it's safe to land.